image_builder/image_builder/config/config.py

```diff
@@ -1,9 +1,9 @@
 from typing import Optional
 
-from pydantic import BaseModel, Extra
+from pydantic import BaseModel
 
 
-class Config(BaseModel, extra=Extra.forbid):
+class Config(BaseModel, extra='forbid'):
```

Comment:
Is there a reason to keep the `extra` parameter? I am not seeing it used anywhere in the below.
https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.extra says that we need a model_config attribute if we plan to use 'forbid'.

Reply:
Is it somehow passed to somewhere?

Reply:
You're right, I see I need to switch to a model_config. It's not passed anywhere, it just changes default validator, so that it doesn't accept redundant fields